packages/expo-router/src/fork/__tests__/getStateFromPath.test.web.ts

```diff
@@ -45,26 +45,30 @@ describe('baseUrl', () => {
     process.env.EXPO_BASE_URL = '/expo/prefix';
 
     const path = '/expo/prefix/bar';
-    const config = configFromFs(['_layout.tsx', 'bar.tsx', 'index.tsx']);
+    const config = getMockConfig({
```

Comment:
It would be nice if this function also accepted an array of strings for brevity.

Reply:
Done & updated the documentation